controllers/accountControllers/userAccountController.js

```diff
@@ -0,0 +1,132 @@
+const {Account} = require('../../models/userAccountModel');
+const {User} = require('../../models/userModel');
+const {validateAccount} = require('../../validation/validationAccount');
+
+
+const createAccount =  async (req, res, next) => {
+    const { error } = await validateAccount.validate(req.body);
+    if(error){
+        return next (new Error(error));
+    }
+    const {
+        accountName, 
+        accountCurrency, 
+        accountAmount, 
+        accountDescription, 
+        accountStat
+    }   = req.body;
+    
+    if (
+        accountName && 
+        accountCurrency && 
+        accountAmount && 
+        accountDescription && 
+        accountStat
+        ) {
+
+        const isAccount = await Account.findOne({accountCurrency});
+
+        if(isAccount){
+            return next (new Error(`${isAccount.accountName} already exists. Please log in!`));
+        }
+
+        const userID = req.user.id;
+        const account = await Account.create({
+            user: userID,
+            accountName, 
+            accountCurrency, 
+            accountAmount, 
+            accountDescription, 
+            accountStat,
+            accountCreated: new Date().toGMTString()
+        });
+    
+        if (account) {
+            res.status(201).json({
+                message: `Your ${account.accountCurrency} account has been successfully created`,
```

Comment:
Do you really need this message? Response `201` is proof that operation completed successfully.

Reply:
I got it. I removed it and try to avoid such repetitions.